### ISSUE TITLE
5.3 Using a pretrained convnet

### ISSUE THREAD

**Issue by 59977** [fchollet/deep-learning-with-python-notebooks#75]
When doing 5.3, the version where you add to the existing convolutional base, I accidentally forgot to set the trainable flag to False. Much to my surprise the model trained fine anyway and pretty much reproduced the results from the book. About 97% accuracy on the test set.

I later realized my mistake and tried again with the flag set to False but then the model trained much worse. Ending up with an accuracy of just 90%. 

This seems weird to me but I can't figure out whats wrong. Has anyone else had this experience?

**Comment by rsnitsch** [comment]
I have the same issue. When working through chapter 5.3, I was only able to reach about 90% accuracy with the VGG16 conv base (trainable set to False). I played around with lots of parameters (epochs, batch size, learning rate, etc) but I was unable to achieve the high accuracy as in the book (the author claims that you can get about 96%).

This has also been reported in the official forums:
https://forums.manning.com/posts/list/45158.page
